core/src/main/java/com/scalar/db/storage/dynamo/DynamoAdmin.java

```diff
@@ -86,18 +332,283 @@ public TableMetadata getTableMetadata(String namespace, String table) throws Exe
 
   @Override
   public Set<String> getNamespaceTableNames(String namespace) throws ExecutionException {
-    throw new UnsupportedOperationException("implement later");
+    try {
+      return metadataManager.getTableNames(namespace);
+    } catch (RuntimeException e) {
+      throw new ExecutionException("getting list of tables failed");
+    }
   }
 
   @Override
   public boolean namespaceExists(String namespace) throws ExecutionException {
-    throw new UnsupportedOperationException("implement later");
+    boolean nameSpaceExists = false;
+    try {
+      ListTablesResponse listTablesResponse = client.listTables();
+      List<String> tableNames = listTablesResponse.tableNames();
+      for (String tableName : tableNames) {
+        if (tableName.startsWith(Utility.getFullNamespaceName(namespacePrefix, namespace))) {
+          nameSpaceExists = true;
+          break;
+        }
+      }
+    } catch (DynamoDbException e) {
+      throw new ExecutionException("getting list of namespaces failed");
+    }
+    return nameSpaceExists;
   }
 
   private String fullNamespace(String namespace) {
     return namespacePrefix.map(s -> s + namespace).orElse(namespace);
   }
 
+  private void controlContinuousBackup(String namespace, String table, boolean noBackup)
+      throws ExecutionException {
+    if (!noBackup) {
+      try {
+        client.updateContinuousBackups(buildUpdateContinuousBackupsRequest(namespace, table));
+      } catch (Exception e) {
+        throw new ExecutionException(
+            "Unable to enable continuous backup for "
+                + Utility.getFullTableName(namespacePrefix, namespace, table));
+      }
+    } else {
+      LOGGER.info(
+          "Continuous backup for "
+              + Utility.getFullTableName(namespacePrefix, namespace, table)
+              + " is disable by default from Dynamo DB");
+    }
+  }
+
+  private PointInTimeRecoverySpecification buildPointInTimeRecoverySpecification() {
+    return PointInTimeRecoverySpecification.builder().pointInTimeRecoveryEnabled(true).build();
+  }
+
+  private UpdateContinuousBackupsRequest buildUpdateContinuousBackupsRequest(
+      String namespace, String table) {
+    return UpdateContinuousBackupsRequest.builder()
+        .tableName(Utility.getFullTableName(namespacePrefix, namespace, table))
+        .pointInTimeRecoverySpecification(buildPointInTimeRecoverySpecification())
+        .build();
+  }
+
+  private void controlAutoScaling(
+      String namespace,
+      String table,
+      boolean noScaling,
+      Set<String> secondaryIndexes,
+      Optional<Long> ru)
```

Comment:
We have the default value of this, so I don't think we need to make it `Optional`.